ui/app/clinical/common/views/visitSummary.html

```diff
@@ -51,9 +51,12 @@
             </div>
             <div ng-switch-when="diagnosis" class="section">
                 <section ng-if="::section.config" class="block dashboard-section section-title has-link">
-                    <bahmni-diagnosis patient-uuid="::patient.uuid" config="::section" show-latest-diagnosis="false"
-                                      show-ruled-out-diagnoses="::section.showRuledOutDiagnoses"></bahmni-diagnosis>
-                    </bahmni-diagnosis>
+                    <h2 ng-hide="::hideTitle" class="section-title" ng-class="::{'hidden-print': (allDiagnoses && allDiagnoses.length == 0)}">
+                        <span> {{::section | titleTranslate}} </span>
+                    </h2>
+                    <bahmni-diagnosis patient-uuid="::section.config.patientUuid"
```

Comment:
@rohit-yawalkar patient.uuid comes from config ? can you please re-check ?

Reply:
yes @binduak, we get patient uuid from either patient object or section.config